lib/index.ts

```diff
@@ -150,6 +262,9 @@ async function buildDepTreeFromFiles(
     throw new Error('Missing required parameters for buildDepTreeFromFiles()');
   }
 
+  const lockFileType = lockFilePath.endsWith('package-lock.json') ?
```

Comment:
This is not specific enough, we have to specifically check for yarn.lock. What if we get passed some other random file name?